packages/sol/src/verification.ts

```diff
@@ -0,0 +1,408 @@
+  const currentYear = new Date().getFullYear();
+  const minYear = currentYear - MAX_USER_AGE;
+  const maxYear = currentYear - MIN_USER_AGE;
+
+  if (userBirthYear < minYear) return FieldError.UserTooOld;
+  if (userBirthYear > maxYear) return FieldError.UserTooYoung;
+  return true;
+}
+
+export function isValidTutorBio(
+  bio: string
+): FieldError.EmptyBio | FieldError.TooLongBio | true {
+  if (!bio.length) return FieldError.EmptyBio;
+  if (bio.length > MAX_TUTOR_BIO_TEXT_LENGTH) return FieldError.TooLongBio;
+
+  return true;
+}
+
+export function isValidTutorAbout(
+  about: string
+):
+  | FieldError.InvalidTutorAbout
+  | FieldError.EmptyTutorAbout
+  | FieldError.TooLongTutorAbout
+  | true {
+  const tutorAboutText = getSafeInnerHtmlText(about);
+
+  if (!tutorAboutText.length) return FieldError.EmptyTutorAbout;
+  if (!HTML_REGEX.test(about)) return FieldError.InvalidTutorAbout;
+  if (tutorAboutText.length > MAX_TUTOR_ABOUT_TEXT_LENGTH)
+    return FieldError.TooLongTutorAbout;
+  return true;
+}
+
+export function isValidTutorNotice(
+  tutorNotice: number
+): FieldError.MaxNoticeExceeded | FieldError.InvalidNotice | true {
+  if (tutorNotice <= MIN_TUTOR_NOTICE_DURATION) return FieldError.InvalidNotice;
+  if (tutorNotice > MAX_TUTOR_NOTICE_DURATION)
+    return FieldError.MaxNoticeExceeded;
+  return true;
+}
+
+export function isValidRuleTitle(
+  title: string
+): FieldError.TooShortRuleTitle | FieldError.TooLongRuleTitle | true {
+  if (title.length > MAX_RULE_TITLE_LENGTH) return FieldError.TooLongRuleTitle;
+  if (title.length < MIN_RULE_TITLE_LENGTH) return FieldError.TooShortRuleTitle;
+  return true;
+}
+
+export function isValidRuleStart(
+  ruleStart: string,
+  ruleEnd: string
+):
+  | FieldError.ISOInvalidRuleStart
+  | FieldError.ISOInvalidRuleEnd
+  | FieldError.RuleStartAfterEnd
+  | FieldError.RuleStartDatePassed
+  | true {
+  if (!Date.parse(ruleStart)) return FieldError.ISOInvalidRuleStart;
+  if (!Date.parse(ruleEnd)) return FieldError.ISOInvalidRuleEnd;
+  if (new Date(ruleStart) > new Date(ruleEnd))
+    return FieldError.RuleStartAfterEnd;
+  if (new Date() > new Date(ruleStart)) return FieldError.RuleStartDatePassed;
+  return true;
+}
+
+export function isValidRuleEnd(
+  ruleEnd: string,
+  ruleStart: string
+):
+  | FieldError.ISOInvalidRuleEnd
+  | FieldError.ISOInvalidRuleStart
+  | FieldError.RuleEndBeforeStart
+  | FieldError.RuleEndDatePassed
+  | true {
+  const currentTime = new Date().getTime();
+  const parsedRuleEnd = Date.parse(ruleEnd);
+  const parsedRuleStart = Date.parse(ruleStart);
+  const ruleEndTime = new Date(ruleEnd).getTime();
+  const ruleStartTime = new Date(ruleStart).getTime();
+
+  if (!parsedRuleEnd) return FieldError.ISOInvalidRuleEnd;
+  if (!parsedRuleStart) return FieldError.ISOInvalidRuleStart;
+  if (ruleEndTime < ruleStartTime) return FieldError.RuleEndBeforeStart;
+  if (ruleEndTime < currentTime) return FieldError.RuleEndDatePassed;
+  return true;
+}
+
```

Comment:
Add a description to the function to indicate that the duration param should be in minutes and not other time units.